Tests/Validator/ValidatorWrapperTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Validator;
+
+use Prophecy\Prophet;
+use Sonata\AdminBundle\Validator\ValidatorWrapper;
+use Symfony\Component\Validator\ConstraintViolationList;
+use Symfony\Component\Validator\ValidatorInterface;
+
+/**
+ * Test for ErrorElement.
+ *
+ * @author Alfonso Machado <email@alfonsomachado.com>
+ *
+ * @group validator-wrapper
+ */
+class ValidatorWrapperTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructor() {
```

Comment:
This will trigger a cs warning. Please type `make cs` and amend your commit with the result.